inc/vendors/classes/class-minify-html.php

```diff
@@ -128,7 +128,10 @@ public function process()
 
         // trim each line.
         // @todo take into account attribute values that span multiple lines.
-        $this->_html = preg_replace('/^\\s+|\\s+$/m', '', $this->_html);
+        // Fixed attribute values which span on multiple lines. Causes double spaces "  "
+        $this->_html = preg_replace('/^\\s+|\\s+$/m', ' ', $this->_html);
+        // Fixed double spaces. Replaced with a single space
+        $this->_html = preg_replace('/  +/', ' ', $this->_html);
```

Comment:
```suggestion
        $this->_html = preg_replace('/\s+/', ' ', $this->_html);
```

@Screenfeed You confirm?

Reply:
Both should work, but I prefer `'/\s+/'` indeed.
What I don't get is the double `\` in `'/^\\s+|\\s+$/m'`. Unless I'm missing something it should be `'/^\s+|\s+$/m'`.

Reply:
Changed from` '/  +/'` into `'/\s+/'`

As for the double `\` in '`/^\\s+|\\s+$/m'`. in  this was in the original file:
https://github.com/mrclay/minify/blob/8e4b0ceb86a4e89bb8031715d1e0f75e953a925f/lib/Minify/HTML.php#L131

I changed into single `\` and tested and is working.

Reply:
I saw was in the original file but I'm surprised it was even working ¯\_(ツ)_/¯
Thanks :)
Is it all fine for you @Tabrisrp?

Reply:
yes all good